### ISSUE TITLE
[BUG] In-house stream - Everything is black

### ISSUE THREAD

**Issue by ruifcardoso** [BrianLima/UWPHook#14]
Hi,

I can launch the game normally using the Steam shortcut. But if I try on a different computer play using stream, the host computer opens the game properly but the stream computer just shows a black screen.

The mouse works, if i move it on the host I can see it moving on the stream computer, and I can also hear the sound. Everything else is black and the stream exits after a few seconds (like 15s or something)

I'm using the lastest version and tested using a few different apps, but what i really want is forza

Thanks

**Comment by BrianLima** [comment]
Have you enabled streaming mode in the settings page?

**Comment by ruifcardoso** [comment]
Yes

**Comment by ruifcardoso** [comment]
Do you need more information?

**Comment by BrianLima** [comment]
Does UWPHook flashes the whole screen on the host computer for a brief moment?

**Comment by ruifcardoso** [comment]
Yes, it "starts" and I can see the mouse synchronized between computers but the background is black. The sound works properly.

I have tried other methods with the same result

Other method tested: https://forum.xda-developers.com/shield-tv/themes-apps/app-launch-windows-8-10-metro-games-t3185985

**Comment by ruifcardoso** [comment]
This isn't probably your problem from what I've been reading.

https://steamcommunity.com/app/333950/discussions/5/617328415057313719/

Tried this and other things but nothing works.

RIP me =(

**Comment by BrianLima** [comment]
I'm afraid it might be something wrong with your Steam instalation or configuration, sorry for that. Steam is having a range of random issues with UWP apps and all we can do is to try to make some hacks to try to make it work better, but the turn around is this, strange bugs and problems. I'll keep trying to make it work, but can't give an ETA tho.

**Comment by BrianLima** [comment]
Hi, I just wrote a woarkaround for this issue on version 2.3, could you please download it and check if it works better now?

**Comment by ruifcardoso** [comment]
It kinda worked a little.

I pressed to launch the game, in this case Forza Horizon 3, old behaviour happened (black screen and then stream quits). But then i tried to open other game, Rainbow Six which also isn't steam (its Uplay), and it didn't work but after a few seconds, Forza appeard and was already on menu (because the game previously was launched and only the stream quits) and it worked (although it was kinda in a teamviewer like window, i could even go to chrome and everything else).

Tried to do the same again, but it didn't work. Launched some other games that have some kind of window before the game (H1Z1 for example) and then forza was there again running.
